app/helpers/tagging_helper.rb

```diff
@@ -15,7 +15,8 @@ def taggable_vita_partners(vita_partners)
   end
 
   def taggable_sites(vita_partners)
-    taggable_vita_partners = vita_partners.sites.includes(:parent_organization).map do |site|
+    taggable_vita_partners = vita_partners.sites.includes(:parent_organization).filter_map do |site|
```

Comment:
this will crash the drop down if there is any site that doesn't have a parent org (aka parent org got deleted at some point); adding this next will avoid this error